state_version.go

```diff
@@ -166,7 +167,7 @@ func (s *stateVersions) Read(ctx context.Context, svID string) (*StateVersion, e
 		return nil, errors.New("invalid value for state version ID")
 	}
 
-	u := fmt.Sprintf("state-versions/%s", url.QueryEscape(svID))
+	u := fmt.Sprintf("state-versions/%s?include=outputs", url.QueryEscape(svID))
```

Comment:
i think, instead of hardcoding the `include` param here, it might be better to add an options struct for any endpoints where you might want to use it (maybe list, read, and current).

you can see an example of this in [organization_membership.go](https://github.com/hashicorp/go-tfe/blob/master/organization_membership.go)

Reply:
Adding this at this point would change the method's signature, which is a breaking change I'd rather avoid.

Given that the outputs are only used by the tests I wrote for StateVersionOutputs support, I think I'll just remove this from state_version.go and implement a new function as a test helper.

Reply:
that sounds totally fine to me, whatever works best for the tests you need to write

you could also potentially follow the pattern used in organization_membership.go where they basically add a new method called ReadWithOptions that takes the options hash. then the original read method just called the ReadWithOptions method with an empty options hash. that might be more work than necessary for your use case though

Reply:
I opted for following the same pattern as organization_membership. Felt like a better option.